src/main/java/com/spikes2212/dashboard/Namespace.java

```diff
@@ -10,45 +15,217 @@
+     * @return a {@link Supplier} with the value set using the {@link NetworkTable}s
+     */
     Supplier<Double> addConstantDouble(String name, double value);
 
+    /**
+     * Adds an integer supplier to the namespace, whose value can only be changed
+     * via the {@link Shuffleboard}, or any other {@link NetworkTable}s UI.
+     *
+     * @param name  the key that will be given to the value
+     * @param value the initial value to be added
+     * @return a {@link Supplier} with the value set using the {@link NetworkTable}s
+     */
     Supplier<Integer> addConstantInt(String name, int value);
 
+    /**
+     * Adds a string supplier to the namespace, whose value can only be changed
+     * via the {@link Shuffleboard}, or any other {@link NetworkTable}s UI.
+     *
+     * @param name  the key that will be given to the value
+     * @param value the initial value to be added
+     * @return a {@link Supplier} with the value set using the {@link NetworkTable}s
+     */
     Supplier<String> addConstantString(String name, String value);
 
-    Namespace addChild(String name);
-
+    /**
+     * Adds a {@link ChildNamespace} to this namespace.
+     *
+     * @param name the name given to the child namespace's name
+     * @return the new {@link ChildNamespace} that is created
+     */
+    ChildNamespace addChild(String name);
+
+    /**
+     * Adds {@link Sendable} data to the namespace.
+     *
+     * @param key   the key that will be given to the value
+     * @param value the value to be added
+     */
     void putData(String key, Sendable value);
 
+    /**
+     * Adds a command to the namespace.
+     *
+     * @param key             the key that will be given to the value
+     * @param command         the command to be added
+     * @param runOnDisable whether the command should be executable when the robot is disabled
+     */
+    default void putCommand(String key, Command command, boolean runOnDisable) {
+        this.putData(key, command.ignoringDisable(runOnDisable));
+    }
+
+    /**
+     * Adds a command that can run on disable to the namespace.
+     *
+     * @param key     the key that will be given to the value
+     * @param command the command to be added
+     */
+    default void putCommand(String key, Command command) {
+        this.putCommand(key, command, true);
+    }
+
+    /**
+     * Adds a runnable value to the namespace that can be run as a {@link InstantCommand}.
+     *
+     * @param key          the key that will be given to the value
+     * @param runnable     the runnable value to be added
+     * @param runOnDisable whether the instant command should be executable when the robot is disabled
+     */
+    default void putRunnable(String key, Runnable runnable, boolean runOnDisable) {
+        this.putData(key, new InstantCommand(runnable).ignoringDisable(runOnDisable));
+    }
+
+    /**
+     * Adds a runnable value that can be run as a {@link InstantCommand} and can run on disable.
```

Comment:
this wording is inconsistent with other places where you describe the ability of a command to run when the robot is disabled -1